<?xml version="1.0"?>
<launch>
    <arg name="is_bag" default="true" />

    <!-- unzipping -->
    <group if="$(arg is_bag)" >
        <include file="$(find point_cloud_object_detector)/launch/unzipping.launch"/>
    </group>

    <!-- point_cloud_object_detector_node -->
    <node pkg="point_cloud_object_detector" type="point_cloud_object_detector_node" name="object_detector" output="screen" >
        <remap from="pc_topic_name" to="/camera/depth_registered/points" />
        <remap from="bbox_topic_name" to="/darknet_ros/bounding_boxes" />
        <remap from="obj_topic_name" to="/object_positions" />
        <remap from="obj_frame_name" to="/base_link" />
        <param name="VOXEL_SIZE" value="0.01" />
    </node>

</launch>